client/plots/disco/viewmodel/ViewModelMapper.ts

```diff
@@ -18,6 +18,7 @@ export class ViewModelMapper {
 		L: 'exonic',
 		Utr3: 'exonic',
 		Utr5: 'exonic',
+		ProteinAltering: 'exonic',
```

Comment:
it should not keep a separate static copy of mclasses. can u change?

Reply:
This is basically a mapping of which mutations are exonic which are non-exonic. Do we have that somewhere else in the code base or in the DB?

Reply:
nope. current fix is fine. sorry